docs/source/create/3d-models/import-model.rst

```diff
@@ -11,9 +11,9 @@ You can import a 3D model into Vircadia that is hosted online or on your domain'
 Import Models from the Cloud
 --------------------------------
 
-If you want your model available to users in other domains, we recommend uploading it to a cloud hosting service of your choice. 
+If you want your model available to users in other domains, we recommend uploading it to a cloud hosting service of your choice.
 
-1. Locate and copy the URL of the model you would like to import. The model should be a OBJ, glTF, or FBX format.
+1. Locate and copy the URL of the model you would like to import. The model should be an FBX, glTF, or OBJ file.
```

Comment:
```suggestion
1. Locate and copy the URL of the model you would like to import. The model should be a glTF, FBX, or OBJ file.
```
To encourage glTF usage.

Reply:
I actually specifically put FBX there to encourage its usage. My understanding is that out FBX support is still better than our glTF support